CHANGELOG.mdx

```diff
@@ -3924,3 +3924,4 @@ Regression: changes from `1.2.12` were missing from `1.4.0`, readded them
 
 * utterance templates defined in actions are checked for existence upon training a new agent, and a warning
   is thrown before training if one is missing
+* added authentication support for rabbitmq url which was missing before
```

Comment:
We don't usually change old changelog entries — the new one needs to be added instead. This guide contains the detailed information about the process: https://github.com/RasaHQ/rasa/blob/main/changelog/README.md

But to keep it short: could you please create a `8057.improvement.md` file inside a `changelog` directory with all the information about what was done in this PR instead of modifying `CHANGELOG.mdx`?

Reply:
@alwx Sounds good!! removed change log entry and created new one as your suggestion

Reply:
Awesome, thanks!